### DISCUSSION TITLE
What I would love to see on the WooCommerce checkout page

### DISCUSSION THREAD

**Discussion by karenwegehenkel** [woocommerce/woocommerce#51717] [Checkout Flow]
For the Woocommerce checkout page, I would love to see an option where the boxes (Contact info/Billing, Shipping, Payment) are hidden like in an accordion with only the main box open and the headlines showing and either a 'next' button, up/down arrows, and/or next box opening automatically, like Shopify does. 
However, this looks great already, and I love all the editing options available for Gutenberg blocks. Well done!

**Comment by mikejolley** [comment]
Hi @karenwegehenkel Thanks so much for taking the time to leave some feedback – much appreciated! 

I had a search and found that a stepped checkout was considered (which is what you describe) but after usability testing we opted to start with the stacked checkout option. Thats not to say we won't add the option to do a stepped checkout in the future for those who prefer this or want to compare both approaches, but right now it isn't a priority.

We make note of all feedback we receive, and we also close issues with the feedback, as active issues are used for bugs or feature/enhancements. It does not mean that we don’t note or review the feedback. Closed issues can also still be commented on.

Again thanks for your comments and we're happy to hear you're a fan of the new blocks.
